emeis/core/views.py

```diff
@@ -57,6 +64,65 @@ class UserViewSet(BaseViewset):
         "city",
     )
 
+    @action(methods=["get"], detail=False)
+    def export(self, request):
+        """Export user list as excel table."""
+        queryset = self.get_queryset()
+        content = list(
+            queryset.values_list(
+                "last_name",
+                "first_name",
+                "email",
+            )
+        )
+        all_acls = models.ACL.objects.filter(user__in=queryset).select_related(
+            "scope", "role", "user"
+        )
+
+        # we use a template to set the headers to bold and align the columns to fit on a DIN-A4 page
+        workbook = openpyxl.load_workbook(settings.USER_EXPORT_TEMPLATE_PATH)
+        sheet = workbook.active
+
+        # set column headers
+        sheet["A1"] = _("Name")
+        sheet["B1"] = _("First Name")
+        sheet["C1"] = _("Email")
+        sheet["D1"] = _("Roles and organizations")
+
+        # get ACLs for each user and format the String according to role and scope
+        acls = []
+        for user in queryset:
+            user_acls = [
+                f"{acl.role.name}: {acl.scope.full_name()}"
```

Comment:
This is very inefficient. Why not loop over `user.acls.all()` instead? Going over all ACLs and checking them for equalling the current user is O(n^2) complexity and turns bad very fast